### DISCUSSION TITLE
Error: ReferenceError: signale is not defined

### DISCUSSION THREAD

**Discussion by mehfooz-ul-haq** [DaggieBlanqx/whatsappcloudapi_wrapper#11] [Q&A]
Hi,
I am trying to setup a webhook, but it throws me an error "**ReferenceError: signale is not defined**" where i try to parse a message from whatsapp on webhook. `await Whatsapp.parseMessage(req.body);` here it throws, can you please help me what can be the possible issue and how to fix it please.

**Comment by farhanali2325** [comment]
add this line in msg_parser.js at the top of the code in node_modules directory of whatsappcloudapi_wrapper

const signale = require('signale')

**Reply by Abdulrehman3802** [reply]
it solved my problem thank you!

**Comment by Abdulrehman3802** [comment]
> Hi, I am trying to setup a webhook, but it throws me an error "**ReferenceError: signale is not defined**" where i try to parse a message from whatsapp on webhook. `await Whatsapp.parseMessage(req.body);` here it throws, can you please help me what can be the possible issue and how to fix it please.

add this line in msg_parser.js at the top of the code in node_modules directory of whatsappcloudapi_wrapper

const signale = require('signale')
